dashboard/test/models/user_test.rb

```diff
@@ -13,6 +12,36 @@ class UserTest < ActiveSupport::TestCase
     assert_equal '21+', teacher.age
   end
 
+  # Disable this test if and when we do require teachers to complete school data
+  test 'school info should not be validated' do
+    school_attributes = {
+      country: 'US',
+      school_type: SchoolInfo::SCHOOL_TYPE_PUBLIC,
+      state: nil
+    }
+    teacher = create :teacher, school_info_attributes: school_attributes
```

Comment:
My belief is that this test could be more succinctly and more clearly written as
```
assert_creates(User)
  create :teacher, school_info_attributes: school_attributes
end
```
Note that `create` performs a `save` (no exclamation mark). Note further that if the validation fails, then `teacher.save!` will throw, meaning the assertion is never reached.